File: src/Link/Link.Frontend/Link-Frontend/src/app/interfaces/event-dto.ts
import { ExpertDto } from './expert-dto';

export interface EventDto {
    id: string;
    userId: string;
    name: string;
    expertType: string;
    status: string;
    latitude: number;
    longitude: number;
    startTime: Date;
    endTime: Date;
    countOfNeededExpert: number;
    experts: ExpertDto[];
}

File: src/Link/Link.ReportManagement.Infrastructure.Models/Models/EventDto.cs
using System;

namespace Link.ReportManagement.Infrastructure.Models.Models
{
    public class EventDto
    {
        public string Id { get; set; }

        public string UserId { get; set; }

        public string Name { get; set; }

        public string ExpertType { get; set; }

        public string Status { get; set; }

        public double Latitude { get; set; }

        public double Longitude { get; set; }

        public DateTime StartTime { get; set; }

        public DateTime? EndTime { get; set; }

        public int CountOfNeededExperts { get; set; }
    }
}
